business_requirement_deliverable_resource_template/models/business_requirement_resource_template.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError
+from openerp.tools.translate import _
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resource_lines(self):
+        rl_data = self.product_id.sudo().resource_lines.copy_data()
+        return [(0, 0, item) for index, item in enumerate(rl_data)]
+
+    @api.multi
+    @api.onchange('product_id')
+    def product_id_change(self):
+        super(BusinessRequirementDeliverable, self).product_id_change()
+        product = self.product_id
+        if product:
+            self.resource_ids = self._prepare_resource_lines()
+        business_requirement = self.business_requirement_id
+        if business_requirement:
+            for resource in self.resource_ids:
+                resource.business_requirement_id = business_requirement.id
+
+
+class BusinessRequirementResourceTemplate(models.Model):
+    _name = "business.requirement.resource.template"
+
+    product_template_id = fields.Many2one(
+        comodel_name='product.template',
+        string='Product',
+        ondelete='set null',
+        copy=False
+    )
+    name = fields.Char('Name', required=True)
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        required=False
+    )
+    uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='UoM',
+        required=True
+    )
+    qty = fields.Float(
+        string='Quantity',
+        default=1,
+    )
+    resource_type = fields.Selection(
+        selection=[('task', 'Task'), ('procurement', 'Procurement')],
```

Comment:
No. I mean create a constant variable in on place to be reused in multiple modules.
The future extension I'm thinking about is a customization module, not a UI table.